DB/sp.sql

```diff
@@ -0,0 +1,36 @@
+
+create or replace procedure sp_changeProfessorName(professor_id int)
+language plpgsql
+as $$
+begin
+    update "Professor" as P
+    set p_name = 'Nombre Creado por SP'
+    where P.p_id = professor_id;
+end;
+    $$;
+
+
+create or replace procedure sp_getStudentPreEnrollments(student_id int)
```

Comment:
Do you think fetching semester offer of the preenrollment as well by joining is a good idea? or Should we make a seperate store procedure to get all semester offers of a pre enrollment and make two seperate calls to the db?

Reply:
Doing a single call is more efficient. I will create a separate SP for this case and later we can delete the ones that are not used